bench/plutus-scripts-bench/plutus-scripts-bench.cabal

```diff
@@ -81,9 +81,9 @@ library
   --------------------------
   build-depends:
     , cardano-api             ^>=9.1
-    , plutus-ledger-api       ^>=1.30
-    , plutus-tx               ^>=1.30
-    , plutus-tx-plugin        ^>=1.30
+    , plutus-ledger-api       ^>=1.30 || ^>= 1.31
```

Comment:
Why do you need this? Is it because of the ghc dependency? If so, Ziyang released 1.31.1 which should fix it.

Reply:
Removed.